src/languageservice/utils/json.ts

```diff
@@ -37,7 +37,7 @@ export function stringifyObject(
       let result = '';
       for (let i = 0; i < obj.length; i++) {
         let pseudoObj = obj[i];
-        if (typeof obj[i] !== 'object') {
+        if (typeof obj[i] !== 'object' || obj[i] === null) {
```

Comment:
fix the null issue when the snippet contains null as a value
it crashed when the snippet looks like this `body: ["string", 5, null]`